pkg/util/selector_test.go

```diff
@@ -310,7 +312,7 @@ func TestClusterMatches(t *testing.T) {
 		want bool
 	}{
 		{
-			name: "cluster excluded",
+			name: "[case 1] cluster excluded",
 			args: args{
```

Comment:
I'd prefer to remove the `args` and use `affinity` instead, we don't need to nest a layer.
I mean as follows:
```
	tests := []struct {
		name     string
		affinity policyv1alpha1.ClusterAffinity
		want     bool
	}
```

Reply:
Structure has been modified